Start/readme2.md

```diff
@@ -1,9 +1,15 @@
 ## 4.3.3   Users Perspective: The Front View 
 The various aspects an users expects from the system and which is my one of the main ideal outcomes of the project can be viewed from the following figure:
 
-![](https://github.com/smriti1313/Property-management-System-Django-Full-Stack-/blob/main/Start/user%20perspective.png)
+<p align=center>
+<img src="user_perspective.png" height=400px >
+</p>
+<br>
+<p align="center">
+  Figure 4.4: Operations offered to the users
+</p>
```

Comment:
@smriti1313 smiriti that's what I told you in previous comment remove ``figure 4.4`` while keep the caption as it is

Reply:
Hi, apologies for the same. Changes has been made.